### PR TITLE
API-36550-response-schema-min-validations

### PR DESCRIPTION
**Pull request by stiehlrod**
## Summary

- Adds changes to rswag spec, and 
- swagger dev (which includes changes from another ticket that wasn't included in the associated PR).

## Related issue(s)

- [API-36551](https://jira.devops.va.gov/browse/API-36551)

## Testing done

- [X] *New code is covered by unit tests*
- [X] Checked local swagger UI

## Screenshots
Screenshot 2024-05-30 at 11 24 55 AM
Screenshot 2024-05-30 at 11 24 47 AM

## What areas of the site does it impact?
	modified:   modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json
	modified:   modules/claims_api/app/swagger/claims_api/v2/production/swagger.json
	modified:   modules/claims_api/spec/requests/v2/veterans/rswag_disability_compensation_request_spec.rb

## Acceptance criteria

- [X]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [X]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

**Review by FonzMP — APPROVED** [review]
:shipit:

File: modules/claims_api/app/swagger/claims_api/v2/production/swagger.json [hunk 1]
```diff
@@ -1912,6 +1912,20 @@
...
+                                    "items": {
+                                      "maxItems": 2,
+                                      "additionalProperties": false,
+                                      "type": "string",
+                                      "enum": [
+                                        "POW",
+                                        "EMP"
+                                      ]
+                                    }
+                                  },
```
Comment (rockwellwindsor-va) [inline modules/claims_api/app/swagger/claims_api/v2/production/swagger.json:1928]: @stiehlrod These are not going out to Production yet, correct?

Comment (stiehlrod) [reply modules/claims_api/app/swagger/claims_api/v2/production/swagger.json:1928]: @rockwellwindsor-va There is a corresponding release note in review as well, so no, waiting until we are ready

Comment (stiehlrod) [reply modules/claims_api/app/swagger/claims_api/v2/production/swagger.json:1928]: @rockwellwindsor-va good catch, thank you!

**Review by rockwellwindsor-va — APPROVED** [review]
LGTM!

**Review by FonzMP — APPROVED** [review]
:shipit:

**Pull request merged**